core/src/apps/benfen/sign_tx.py

```
@@ -0,0 +1,235 @@
+                if referenced_command["type"] == "SplitCoin":
+                    amount_input_index = referenced_command["data"]["amounts"][0][
+                        "index"
+                    ]
+
+    if amount_input_index is None or recipient_input_index is None:
+        raise wire.DataError("Required commands not found")
+
+    try:
+        amount_hex = inputs[amount_input_index]["Pure"]
+        amount_raw = int.from_bytes(binascii.unhexlify(amount_hex), "little")
+    except Exception:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)